homeassistant/components/homewizard_climate/climate.py

```diff
@@ -0,0 +1,226 @@
+"""Climate platform for homewizard_climate."""
+import logging
+import time
+
+from homewizard_climate_websocket.model.climate_device_state import (
+    HomeWizardClimateDeviceState,
+)
+from homewizard_climate_websocket.ws.hw_websocket import HomeWizardClimateWebSocket
+
+from homeassistant.components.climate import (
+    FAN_HIGH,
+    FAN_LOW,
+    FAN_MEDIUM,
+    FAN_OFF,
+    FAN_ON,
+    SWING_HORIZONTAL,
+    SWING_OFF,
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Todo."""
+    websockets = hass.data[DOMAIN][entry.entry_id]["websockets"]
+    entities = [HomeWizardClimateEntity(ws, hass) for ws in websockets]
+    async_add_entities(entities)
+
+
+class HomeWizardClimateEntity(ClimateEntity):
```

Comment:
To allow more climate types in the future, it is maybe nice to create a class for the Heater Fan (with all its options) and a class for others. But this works for now

Reply:
Great suggestion. At this point I don't even know what the common denominator is between different devices from the cloud provider. I will make sure to do so, as soon as a second device type needs to be added.

Reply:
Definitely, I hit you up in a DM on Discord. 👍